src/st/strategy/shortcode/class-wpml-pb-shortcode-content-wrapper.php

```diff
@@ -250,4 +255,62 @@ private function insert_wrappers() {
 			$offset          = $offset + mb_strlen( $wrapper );
 		}
 	}
+
+	/**
+	 * @param string $content
+	 * @param array  $shortcodes
+	 *
+	 * @return string
+	 */
+	public static function maybeWrap( $content, array $shortcodes ) {
+		/** @see WPML_Gutenberg_Integration::GUTENBERG_OPENING_START */
```

Comment:
@strategio Why have this comment when you can code it:
```php
$notGutenbergContent  = pipe( Str::includes( WPML_Gutenberg_Integration::GUTENBERG_OPENING_START ), Logic::not() );
```

Reply:
@brucepearson, the class `WPML_Gutenberg_Integration` is in another project, that's why I opted for a PHPDoc. Otherwise I would have to create a stub which is overkill just for this.

Reply:
@strategio So the comment confused me more than helping. Maybe it's better to just add a duplicate class constant in this class.